ref/general/microshed.adoc

```diff
@@ -6,29 +6,34 @@
 :seo-description:  MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 = Testing in a container
 
-MicroShed testing helps you to write integration tests by using Testcontainers for Java microservice applications. With MicroShed testing you test your application in the same container that you use in production, thus reducing the risk of test failure.
+MicroShed testing helps you to write integration tests for Java microservice applications.
+With MicroShed testing, you reduce the risk of test failure by testing your application in the same container that you use in production.
 
-Integration testing looks at microservices as a group and tests them as a whole unit. Intergration testing offers advantages over other types of testing, for example, detecting bugs earlier in the cycle.
+Integration testing looks at microservices as a group and tests them as a whole unit.
+Intergration testing offers advantages over other types of testing, for example, detecting bugs earlier in the cycle.
 
 == Why integration testing
 
-In integration testing, you can put together individual pre-tested modules and test them as a group. Integration testing follows unit testing. The modules used are previously unit tested, but now they are now tested as a whole to evaluate compliance and functionality.
-
+Integration testing follows unit testing.
+After individual modules are unit tested, integration testing looks at the modules together to evaluate their performance as a group.
 
 == What challenges does MicroShed resolve
 
-An application might run differently in production as compared to test/dev environment. This can cause a test to fail in production though it passed in  dev/test. You can address this issue by testing your application in the same container that you’ll use in production.
+An application might run differently in production as compared to the development and testing environment.
+This can cause a test to fail in production though it passed in  development.
```

Comment:
"This can cause a test to fail in production though it passed in  development"

"Differences between the environments can cause a test to fail in production, even though it passed in development."

Avoid "This" as subject, remove extra space before "development"